instrumentation/opentelemetry-instrumentation-flask/setup.cfg

```diff
@@ -49,6 +49,7 @@ install_requires =
 [options.extras_require]
 test =
     opentelemetry-test-utils == 0.28b1
+    markupsafe==2.0.1
```

Comment:
Curious as to what the breaking change was and why these specific instrumentations need to be pinned?

Reply:
Probably [this one](https://github.com/open-telemetry/opentelemetry-python/issues/2476).

Reply:
https://github.com/open-telemetry/opentelemetry-python-contrib/runs/5265920492?check_suite_focus=true

```
 _ ERROR collecting instrumentation/opentelemetry-instrumentation-flask/tests/test_programmatic.py _
ImportError while importing test module '/home/runner/work/opentelemetry-python-contrib/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-flask/tests/test_programmatic.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test_programmatic.py:17: in <module>
    from flask import Flask, request
../../../.tox/py37-test-instrumentation-flask/lib/python3.7/site-packages/flask/__init__.py:14: in <module>
    from jinja2 import escape
../../../.tox/py37-test-instrumentation-flask/lib/python3.7/site-packages/jinja2/__init__.py:12: in <module>
    from .environment import Environment
../../../.tox/py37-test-instrumentation-flask/lib/python3.7/site-packages/jinja2/environment.py:25: in <module>
    from .defaults import BLOCK_END_STRING
../../../.tox/py37-test-instrumentation-flask/lib/python3.7/site-packages/jinja2/defaults.py:3: in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
../../../.tox/py37-test-instrumentation-flask/lib/python3.7/site-packages/jinja2/filters.py:13: in <module>
    from markupsafe import soft_unicode
E   ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/home/runner/work/opentelemetry-python-contrib/opentelemetry-python-contrib/.tox/py37-test-instrumentation-flask/lib/python3.7/site-packages/markupsafe/__init__.py)
```